docs/concepts.md

```diff
@@ -1,37 +1,37 @@
 Concepts
 ========
 
-Simplistic Overview of request processing
+Overview of request processing
 -----------------------------------------
 
-A request is received on the edge server (fastly) and is associated to a service configuration.
-The configuration contains a varnish cache script (VCL) that extracts processing information from
-the request. those are:
+An HTTP request is received on the edge server (fastly) and is associated to a [service configuration][fastly-services].
+The configuration contains a varnish cache script ([VCL][vcl]) that extracts processing information from
+the request. Those are:
 
 - requested resource
 - selector and extension
```

Comment:
We should define what is meant by selector and extension here - I think extension means "file extension"?

Reply:
selector is the URL part between the resource and the extension, e.g. /foo.**bar**.html

Reply:
What purpose does the selector serve? I will use this as a start for a definitions section but I think we need more.

Reply:
You can use selectors to render different variations of the same underlying resource. For example, you might choose to have an `/index.html` (default/empty selector), an `/index.toc.json` for a table of contents of the same content in JSON format, or an `/index.sitemap.xml` etc. developer.adobe.com is also using a bunch of selectors.